core/src/test/scala/unit/kafka/server/CreateTopicsRequestWithForwardingTest.scala

```diff
@@ -26,14 +26,19 @@ import scala.jdk.CollectionConverters._
 
 class CreateTopicsRequestWithForwardingTest extends AbstractCreateTopicsRequestTest {
 
-  override def enableForwarding: Boolean = true
+  override def enableForwarding: Boolean = if (isKRaftTest()) true else false
 
   @ParameterizedTest
   @ValueSource(strings = Array("zk", "kraft"))
```

Comment:
I think we can make this kraft only for now until we support forwarding in zk clusters. Otherwise it is a little weird to have a test called `testForwardToController` which does not do any forwarding.